### DISCUSSION TITLE
Form Button Doesn't Have Box Shadow Option

### DISCUSSION THREAD

**Discussion by calebthomasg** [elementor/elementor#23281] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When you add a form widget to a page, there is no option to add a box shadow to the button attached to the form like you can do on the stand-alone button. 

### Describe the solution you'd like

Allow the option to add a drop shadow on a form button. 

Also, adding the option to display a drop shadow on hover. I've attached an example of that below.

https://github.com/elementor/elementor/assets/76978272/e274bb8d-c709-4362-a3f6-2f85cf549a31

### Describe alternatives you've considered

I've created a form using the Contact Form 7 plugin in WordPress. Then I added classes inside the form and styled them using CSS.

**Comment by brandonecarr** [comment]
Under the Content tab, expand the Buttons dropdown. Inside, there is a Button ID field. Make up an ID (formButton), and then in the Custom CSS section simply add: 

#formButton {
  color: #000;
}
